src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/HmrcPageHeadingSection.scala.html

```diff
@@ -0,0 +1,20 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(section: String, as: String = "p")
```

Comment:
Would we ever want this used on its own? I wonder if we should make it package private if not.

Reply:
we probably wouldn't and I've just realised something else in reviewing this, that we have inline copy here that's not translatable. So I guess it definitely needs to shift around a bit

- [ ] revisit how the section is being output so it's more hidden and translatable

Reply:
Be worth making the "This section is" optionally overwritable with something custom. That text is not always suitable.